hugolib/sitemap_test.go

```diff
@@ -86,11 +86,12 @@ func doTestSitemapOutput(t *testing.T, internal bool) {
 
 func TestParseSitemap(t *testing.T) {
 	t.Parallel()
-	expected := Sitemap{Priority: 3.0, Filename: "doo.xml", ChangeFreq: "3"}
+	expected := Sitemap{ChangeFreq: "3", Priority: 3.0, Filename: "doo.xml", Exclude: true}
```

Comment:
Exclude was added here but I also changed the order of the properties to match the order they appear below and in Docs.